sample/test.yml

```diff
@@ -8,7 +8,7 @@
 # Enable printing debug information during testing
 # Default value is True.
```

Comment:
In vars/test.yml, sample/test.yml and vars/create_windows_template_vars.yml, I think all True/False in comments can be updated to lower case